### DISCUSSION TITLE
What unique traces can be used to identify a Skeleton?

### DISCUSSION THREAD

**Discussion by ceopeo** [skeletonlabs/skeleton#2351] [General]
Hey guys. What traces can be used to identify Skeleton through the page sources?

**Comment by endigo9740** [comment]
Hey @ceopeo, do you mean checking if a project is built with Skeleton? If you have access to the project's `package.json` that would be the simplest ways to confirm. Otherwise I'd suggest inspecting button classes. If it has something like `btn variant-filled-primary` that would help confirm.

**Reply by ceopeo** [reply]
Hey mate, thank you!!
![image](https://github.com/skeletonlabs/skeleton/assets/3456008/c2318549-c0b5-43bc-8753-1ee90e533f24)
https://phpcodechecker.com/ Found this example, I need unique traces. This one doesn't fit a bit
